src/TestFramework/Factory.php

```diff
@@ -27,7 +27,7 @@
     /**
      * @var string
      */
-    private $tempDir;
+    private $tmpDir;
```

Comment:
I like that we are making this consistent all over the code base, but the diff of this PR is already quite big, so maybe this should be moved to a separate PR. That would make it a bit easier to see exactly what changed.

Reply:
I will spend more time creating separate PR for these 2 files than review requires :) leaving it here